libs/components/core/src/lib/modules/shared/number-format/number-format-utility.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('Number format utility', function () {
     const digits = '0.9-0';
 
     expect(() => formatCurrency(50, digits)).toThrowError(
-      'minimumIntegerDigits value is out of range.'
+      /^(minimumIntegerDigits value is out of range\.|invalid digits value: 0)$/
```

Comment:
Firefox has different text in its error message; the error comes from [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat).